tICA/scripts/ComputeGroupTICA.m

```diff
@@ -0,0 +1,247 @@
+function ComputeGroupTICA(StudyFolder, SubjListName, TCSListName, SpectraListName, fMRIListName, sICAdim, RunsXNumTimePoints, TCSConcatName, TCSMaskName, AvgTCSName, sICAAvgSpectraName, sICAMapsAvgName, sICAVolMapsAvgName, OutputFolder, OutString, RegName, LowResMesh, tICAmode, tICAMM)
+    
```

Comment:
No usage statement/help for the function.

Reply:
Resolving these, as these are not intended to be user-interfacing functions.  The script that calls them is slightly more user-facing, but still an internal script, not intended to be called directly anyway.